app/controllers/howlers_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HowlersController < ApplicationController
+  def index
+    @howlers = current_user.howlers.order(:created_at).reverse_order
```

Comment:
I should probably create a class method to do the ordering, just to push the logic down the stack.